app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">Weather</string>
+    <string name="title_current">Current</string>
+    <string name="title_forecast">Forecast</string>
+    <string name="title_history">History</string>
+    <string name="title_o">o</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_o` appears to be unused
 Unused resources make applications larger and slow down builds.